src/Cool/DidYouKnow/didYouKnowModal.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <string>
+#include <vector>
+#include "cereal/types/chrono.hpp"
+class DidYouKnowModal {
+public:
+    DidYouKnowModal() = default;
```

Comment:
I don't think there is a need to even mention the default constructor. It can be implicitly generated.